reference/channels.v3.yml

```diff
@@ -1930,6 +1934,12 @@ components:
         title: A deleted channel cannot be updated. Channel can be restored within 90 days after deleting by contacting BigCommerce support team
         type: /api-docs/getting-started/api-status-codes
         errors: {}
+    error_422_invalid_channel_name_resp_example:
+      value:
+        status: 422
+        title: Only latin alphabet letters, arabic numerals, dash and underscore are allowed to be used as channel name.
```

Comment:
```suggestion
        title: You can only use Latin alphabet letters, Arabic numerals, dashes, and underscores as channel names.
```

Reply:
I'll have to keep this message as is because it's the actual error message. Hopefully processes can be changed so we can write the error messages in the future

Reply:
Oh okay

Reply:
@katerinagl If you could change the error message on your end, I can update the docs accordingly